$OpenBSD: patch-iodev_network_slirp_slirp_h,v 1.1 2014/11/04 06:01:41 brad Exp $

slirp/cksum.cc:83: error: 'uintptr_t' was not declared in this scope

slirp/compat.cc:141: error: 'WIFEXITED' was not declared in this scope
slirp/compat.cc:144: error: 'WEXITSTATUS' was not declared in this scope

--- iodev/network/slirp/slirp.h.orig	Tue Nov  4 00:14:00 2014
+++ iodev/network/slirp/slirp.h	Tue Nov  4 00:19:24 2014
@@ -33,6 +33,8 @@ typedef char *caddr_t;
 #endif
 
 #include <sys/types.h>
+#include <stdint.h>
+#include <sys/wait.h>
 #ifdef HAVE_SYS_BITYPES_H
 # include <sys/bitypes.h>
 #endif
